src/docs/cookbook/testing/unit/introduction.md

```diff
@@ -108,7 +108,7 @@ from the `test` package.
 <!-- skip -->
 ```dart
 // Import the test package and Counter class
-import 'package:test/test.dart';
+import 'package:flutter_test/flutter_test.dart';
```

Comment:
Ah, good catch! One Q: Instead of changing this line, should we encourage folks to import the `test` package directly? Since this is related to unit tests, I was hoping to keep it pure Dart!

Reply:
> this is related to unit tests, I was hoping to keep it pure Dart!

That was my feeling too.